src/dfx/Cargo.toml

```diff
@@ -57,7 +57,8 @@ ic-agent = { workspace = true, features = ["reqwest"] }
 ic-asset.workspace = true
 ic-identity-hsm = { workspace = true }
 ic-utils = { workspace = true }
-ic-wasm = "0.2.1"
+# FIXME set this to 0.3.6 when it is released
```

Comment:
Is this release imminent?  Or will we be holding this PR open for some time while we await ic-wasm 0.3.6?

Reply:
Release is imminent. I was just asking Yan to hold off on it while this PR was being worked on in case some changes needed to be made